src/recursion.js

```diff
@@ -0,0 +1,24 @@
+module.exports = function recursion(obj) {
+    if(typeof obj == "object"){
+    let arr = [];
+    function f(lOrR, stage = 0) {
```

Comment:
Please, don't use such naming `lorR`, you should name your variables in readable and maintainable for others